config/samples/httpbin_protected.yaml

```diff
@@ -8,6 +8,8 @@ spec:
   name: httpbin protected
   protocol: http
   active: true
+  jwt_default_policies:
+    - "default/httpbin"
```

Comment:
This looks like an artifact, please revert as this example is not a JWT api.

Reply:
done 2badfc2